packages/components/src/range-control/index.js

```diff
@@ -42,7 +42,7 @@ function RangeControl(
 		beforeIcon,
 		className,
 		currentInput,
-		color: colorProp = color( 'ui.brand' ),
+		color: colorProp = color( 'ui.theme' ),
```

Comment:
Is there any reason why not all use cases have fallbacks like this one?

Reply:
This value is  (eventually) passed into the `styled` generated component, which has a fallback:
https://github.com/WordPress/gutenberg/pull/24059/files#diff-bc2454ef211a3ab799a3f1c4dfe936c1R36

This workflow isn't ideal 🤦‍♀️ . I'm looking into an automated solution now.
I have something in mind 🤞

Reply:
Agreed that automating would be good or at least a lint rule. But I'm willing to include this fix quickly if you think it's ready as is?

Reply:
It is ready as is 👍